doc/kubernetes/modules/ROOT/pages/running/switch-back.adoc

```diff
@@ -0,0 +1,115 @@
+
+include::partial$running/infinispan-batchcr-intro.adoc[]
+
+The `ConfigMap` with the {ispn} CLI operations are as follows:
+
+.Checks the cross-site status for all caches backing up to `{site-a}`.
+[source,yaml]
+----
+include::example$helm/ispn-site-b.yaml[tag=infinispan-crossdc-status]
+----
+
+.Starts a cross-site state transfer for all caches backing up to `{site-a}`.
+[source,yaml]
+----
+include::example$helm/ispn-site-b.yaml[tag=infinispan-crossdc-push-state]
+----
+
+.Checks the cross-site state transfer progress.
+[source,yaml]
+----
+include::example$helm/ispn-site-b.yaml[tag=infinispan-crossdc-push-state-status]
+----
+
+.Resets the cross-site state transfer progress after its completion.
+[source,yaml]
+----
+include::example$helm/ispn-site-b.yaml[tag=infinispan-crossdc-reset-push-state-status]
+----
+
+.Clears the cache data in `{site-a}`.
+[source,yaml]
+----
+include::example$helm/ispn-site-a.yaml[tag=infinispan-crossdc-clear-caches]
+----
+
+[#running-infinispan-batch]
+==== Running {ispn} Batch
+
+include::partial$running/infinispan-running-batch.adoc[]
+
+==== Procedures
+
+After the {ispn} in the primary site is back online and joined the cross-site channel (see xref:running/infinispan-crossdc-deployment.adoc#verifying-the-deployment[verifying the {ispn} deployment]), the state transfer must be manually started from the secondary site.
+
+After clearing the state in the primary site, it transfers the full state from the secondary site to the primary site, and it must be completed before the primary site can start handling incoming requests.
+
+WARNING: Transferring the full state may impact the {ispn} cluster perform by increasing the response time and/or resources usage.
+
+The first procedure is to delete any stale data from the primary site.
+
+. Login to the primary site.
+
+. Shutdown Keycloak. This will clear all Keycloak caches, and it prevents the state of Keycloak from being out-of-sync with Infinispan.
+
+. Run the `crossdc-clear-caches` Batch CR (see xref:#running-infinispan-batch[]).
+
+. Delete all Batch CRs created.
+
+Now we are ready to transfer the state from the secondary site to the secondary site.
+
+. Login into your secondary site.
+
+. Run the `crossdc-push-state-status` Batch CR (see xref:#running-infinispan-batch[]).
+
+. Check the replication status is `ONLINE` for all caches with `crossdc-status` Batch CR.
+
+. Wait for the state transfer to complete by checking the output of the Batch CR `crossdc-push-state-status`.
+
+. If an error is reported by step 4, try to repeat the operations from step 2.
+
+. Clear the state transfer status with Batch CR `crossdc-reset-push-state-status`
+
+. Delete all Batch CRs created.
+
+. Login to the primary site.
+
+. Start Keycloak.
+
+Both {ispn} clusters are in sync and the switchover from secondary back to the primary site can be performed.
+
+=== AWS Aurora Database
```

Comment:
Should we state that this is optional?

Reply:
I'm not sure which paragraph you've pointing your comment to, could you please elaborate?

Reply:
The switching back of the Aurora DB writer from one AZ to another. Long term there will be a performance benefit of having the writer in the same AZ as the Keycloak primary, however this comes at the expense of more downtime when recovering. So I was thinking this step could be optional as it will improve perf, but not affect data consistency.

Maybe it's simpler to just state that this is necessary so that the docs are more focused.

Reply:
Thank you. I'll add the following sentence in the next commit: 

> SSwitching the writer instance of Aurora will lead to a small downtime, and having the writer instance in the other datacenter with a slightly longer latency might be acceptable for some deployments.
So this might be deferred to a maintenance window or skipped depending on the circumstances of the deployment.